php/meterpreter/ext_server_stdapi.php

```diff
@@ -1267,6 +1268,40 @@ function stdapi_registry_set_value($req, &$pkt) {
 }
 }
 
+if (!function_exists('stdapi_net_config_get_interfaces')) {
+if (is_callable('net_get_interfaces')) {
+    register_command('stdapi_net_config_get_interfaces', COMMAND_ID_STDAPI_NET_CONFIG_GET_INTERFACES);
+}
+function stdapi_net_config_get_arp_interfaces($req, &$pkt) {
```

Comment:
I think this needs to be: `stdapi_net_config_get_interfaces`

Reply:
@dledda-r7?